webpack/assets/javascripts/react_app/components/AutoComplete/AutoCompleteActions.js

```diff
@@ -99,16 +136,21 @@ const requestSuccess = ({
   });
 };
 
-const requestFailure = ({ error, id, dispatch }) =>
+const requestFailure = ({ error, id, dispatch, isVisible = true }) => {
+  if (!isVisible) {
+    // eslint-disable-next-line no-console
+    console.error('[Autocomplete]:', error);
+  }
   dispatch({
     type: AUTO_COMPLETE_FAILURE,
     payload: {
       results: [],
-      error: error.message || error,
+      error: isVisible ? error : null,
```

Comment:
Instead of `console.error`, would it make more sense to dispatch payload of `{ error, isVisible }` so it always logs and stores the error?

A selector can select the error only if it should be visible.

Reply:
thanks, I like it better than using the console